src/EnergyPlus/HVACVariableRefrigerantFlow.hh

```diff
@@ -749,6 +749,8 @@ namespace HVACVariableRefrigerantFlow {
         int coolCoilAirOutNode;             // cooling coil air outlet node number
         int heatCoilAirInNode;              // heating coil air inlet node number
         int heatCoilAirOutNode;             // heating coil air outlet node number
+        bool coolingCoilActive;             // turns ON or OFF the cooling coil in the TU depending zone load when parent HP is in cooling mode
```

Comment:
We are also moving to initilizing here and deleting the constuctor. You could have initilized this var to false here and left the constructor alone.

Reply:
done.